tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -0,0 +1,25 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const httpHelper = require('../helpers/httpHelper')
+const fetch = require('node-fetch')
+
+When('the user is sent a notification', function () {
+  const headers = httpHelper.createAuthHeader(client.globals.backend_admin_username)
+  return fetch(
+    client.globals.backend_url + '/ocs/v2.php/apps/testing/api/v1/notifications',
+    { method: 'POST', headers: headers }
```

Comment:
send the user name here. e.g. you can specify it as parameter in the step line. You might want to check that the notification is displayed to the correct user. If the notification is send to `user2` will `user1` see it?